nextflow_schema.json

```diff
@@ -10,25 +10,57 @@
             "type": "object",
             "fa_icon": "fas fa-terminal",
             "description": "Define where the pipeline should find input data and save output data.",
-            "required": [
-                "input"
-            ],
+            "required": ["input", "outdir"],
             "properties": {
                 "input": {
                     "type": "string",
                     "format": "file-path",
+                    "exists": true,
+                    "schema": "assets/schema_input.json",
                     "mimetype": "text/csv",
                     "pattern": "^\\S+\\.csv$",
-                    "schema": "assets/schema_input.json",
                     "description": "Path to comma-separated file containing information about the samples in the experiment.",
-                    "help_text": "You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row. See [usage docs](https://nf-co.re/magmap/usage#samplesheet-input).",
+                    "help_text": "You will need to create a design file with information about the samples in your experiment before running the pipeline. Use this parameter to specify its location. It has to be a comma-separated file with 3 columns, and a header row.",
                     "fa_icon": "fas fa-file-csv"
                 },
-                "outdir": {
+                "genomeinfo": {
                     "type": "string",
-                    "description": "Path to the output directory where the results will be saved.",
-                    "default": "./results",
-                    "fa_icon": "fas fa-folder-open"
+                    "format": "file-path",
+                    "mimetype": "text/tsv",
+                    "fa_icon": "fas fa-dna",
+                    "description": "Path to comma-separated file containing information about the genomes.",
+                    "schema": "assets/schema_genomeinfo.json"
+                },
+                "gtdbtk_metadata": {
+                    "type": "string",
+                    "description": "Path to comma-separated file containing the output from GTDB-Tk.",
+                    "fa_icon": "fas fa-bacteria"
+                },
+                "ncbi_genome_infos": {
+                    "type": "string",
+                    "default": "./assets/ncbi_genome_infos.csv",
+                    "description": "Path to txt file with information about genomes in NCBI.",
```

Comment:
What information?

Throughout this whole schema please add the longer-form help desk and be  much more specific what goes into each parameter